Mollie.Api/Models/Payment/PaymentMethod.cs

```diff
@@ -21,5 +21,6 @@ public static class PaymentMethod {
         public const string ApplePay = "applepay";
         public const string MealVoucher = "mealvoucher";
         public const string In3 = "in3";
+        public const string POS = "pointofsale";
```

Comment:
In general I try not to use abbreviations. So in this case the name of the property would be "PointOfSale"

Reply:
Hey Vincent, just updated it.